python/anvill/binja.py

```diff
@@ -131,8 +131,9 @@ def _collect_xrefs_from_inst(bv, inst, ref_eas, reftype=XrefType.XREF_NONE):
     if not isinstance(inst, bn.LowLevelILInstruction):
         return
 
-    assert not is_unimplemented(bv, inst)
-    assert not is_undef(bv, inst)
+    if (is_unimplemented(bv, inst)
```

Comment:
get rid of extraneous `(` and `)` and move the `or ..` onto the same line as `is_unim...`